storage/tick.go

```diff
@@ -191,6 +191,9 @@ func (mgr *tickManager) Tick(forceType forceType) error {
 			break
 		}
 		mgr.sleepFn(interval)
+		// Check again at the end of each sleep to see if it
+		// has changed.
```

Comment:
Do you want to put a note here about this being useful for tests?

Reply:
sure

Reply:
done